sql/pg_duckdb--0.0.1.sql

```diff
@@ -117,20 +123,22 @@ $func$;
 -- iceberg_* functions optional parameters are extract from source code;
 -- https://github.com/duckdb/duckdb_iceberg/tree/main/src/iceberg_functions
 
-CREATE OR REPLACE FUNCTION iceberg_scan(path text, allow_moved_paths BOOLEAN DEFAULT FALSE,
+CREATE FUNCTION @extschema@.iceberg_scan(path text, allow_moved_paths BOOLEAN DEFAULT FALSE,
                                                    mode TEXT DEFAULT '',
                                                    metadata_compression_codec TEXT DEFAULT 'none',
                                                    skip_schema_inference BOOLEAN DEFAULT FALSE,
                                                    version TEXT DEFAULT 'version-hint.text',
                                                    version_name_format TEXT DEFAULT 'v%s%s.metadata.json,%s%s.metadata.json')
-RETURNS SETOF record LANGUAGE 'plpgsql' AS
+RETURNS SETOF record LANGUAGE 'plpgsql'
+SET search_path = pg_catalog, pg_temp
+AS
 $func$
 BEGIN
     RAISE EXCEPTION 'Function `iceberg_scan(TEXT)` only works with Duckdb execution.';
 END;
 $func$;
 
-CREATE TYPE iceberg_metadata_record AS (
+CREATE TYPE duckdb.iceberg_metadata_record AS (
```

Comment:
Same question here (and below where we have `duckdb.xx`

Reply:
We could, but it pollutes the `public` namespace with stuff that does not need to be there. We mainly put a few functions in the public schema to allow accessing them without the `duckdb.` prefix. These types we no-one really needs to reference, so there's no reason for them to be in the `public` schema.